db/migrate/20180917082341_add_conversion_host_limits.rb

```diff
@@ -0,0 +1,11 @@
+class AddConversionHostLimitsColumns < ActiveRecord::Migration[5.0]
```

Comment:
This class doesn't match the name of the file, did you create this with `rails generate migration AddConversionHostLimits` or create the file manually?

Reply:
I've redone the migration using rails generator. Now tests are green.

Reply:
:+1: recommend squashing those commits since the first doesn't add any value to the history